### ISSUE TITLE
dataset error

### ISSUE THREAD

**Issue by jala1494** [monjurulkarim/ROL_Dataset#3]
In ROL dataset , There are some error.
These 'toa' is over 100. it have to less than 100.
    9452_5.npz,2661
    monjurul2651.npz,126
    monjurul3854.npz,100
    monjurul298.npz,100
    monjurul6268.npz,132
    monjurul5621.npz,120
    monjurul8979_2.npz,313
    monjurul7294.npz,100
    9452_2.npz,1038
    2892.npz,213
    monjurul5018.npz,115
    monjurul2224.npz,172
    monjurul8662.npz,106

please correct ROL data set as in your dataset

**Comment by monjurulkarim** [comment]
@jala1494 Thank you for your interest in our dataset.

Upon a closer look at our training code, you may notice that we did not incorporate 'toa' in training AM-Net. I'd like to clear up a potential misunderstanding: the feature files (.npz) are specifically related to the extracted CNN features (RGB/flow). The 'toa' values in the npz files are predominantly random, as they do not factor into the training loop. For assessing performance in terms of earliness, we have provided a separate csv file named `val_toa.csv`, which contains the time of accident (toa). You can now locate it in the shared drive.

To replicate our results, please start by using the `demo.py` script to generate the riskiness score. Then, utilize our evaluation metric to generate results using the obtained riskiness scores. For your convenience, we have also included our pre-trained weights.

We extend our sincere apologies for any oversight in not explicitly mentioning this in the dataset description

**Comment by jala1494** [comment]
Thank you for replying.
